modules/ballerina-native/src/test/resources/samples/datatableTest.bal

```diff
@@ -374,9 +370,9 @@ function testColumnAlias () (int i, int l, float f, float d, boolean b, string s
 }
 
```

Comment:
Put a test to include the negative case of not finding a specific database type. We can pass in a invalid database type and put a JDBC URL to point to HSQLDB.